frontend/src/views/activity/Activity.vue

```diff
@@ -103,15 +103,14 @@ Displays a single activity
               </div>
             </div>
           </v-card>
-          <component :is="'ActivityLayout' + activityType.template" v-if="!activityType._meta.loading" :activity="activity" />
+          <component :is="'ActivityLayoutGeneral'" :activity="activity" />
```

Comment:
Das könntest du vereinfacht als `<activity-layout-general :activity="activity" />` schreiben. Ich weiss dass wir in #834 noch über Layouts sprechen, aber es wird schlussendlich kein grosser Aufwand sein das `:is` wieder reinzumachen.

Reply:
Ich finde das eine gute Idee. Würde aber dann lieber gerade so machen:
```suggestion
          <activity-layout type="general" :activity="activity" />
```

Reply:
ActivityLayout gibt es heute nicht.
Beinhaltet dein Vorschlag, dass eine neue Vue-Komponente programmiert wird?

Reply:
Für beide vorgeschlagenen Varianten müsstest du nur die Komponente components/activity/layouts/General.vue umbenennen, zu ActivityLayoutGeneral bzw. zu ActivityLayout. Mehr Funktionalität wie zB. Unterstützung für weitere `type`s sollte meiner Meinung nach noch nicht vor #834 programmiert werden.

Reply:
Habe mich für ActivityContentLayout entschieden... da es "nur" die ActivityContents zeigt (nicht die ganze Activity).